docs/execution_providers/OpenVINO-ExecutionProvider.md

```diff
@@ -20,14 +20,15 @@ session.set_providers(['OpenVINOExecutionProvider'], [{Key1 : Value1, Key2 : Val
 *Note that this causes the InferenceSession to be re-initialized, which may cause model recompilation and hardware re-initialization*
 
 ### C/C++ API
-All the options (key-value pairs) need to be concantenated into a string as shown below and passed to OrtSessionOptionsAppendExecutionProviderEx_OpenVINO() API as shown below:-
+All the options shown below are passed to SessionOptionsAppendExecutionProvider_OpenVINO() API and populated in the struct OrtOpenVINOProviderOptions in an example shown below:-
 
 ```
-std::string settings_str;
-settings_str.append("Key1|Value1\n");
-settings_str.append("Key2|Value2\n");
-settings_str.append("Key3|Value3\n");
-Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProviderEx_OpenVINO(sf, settings_str.c_str()));
+OrtOpenVINOProviderOptions options;
+options.device_type = "CPU_FP32";
+options.enable_vpu_fast_compile = 0;
```

Comment:
Let us add a comment so that user/developer doesn't get confused if he has to use this option using c++ API
options.enable_vpu_fast_compile = 0;      // 0 = false, nonzero = true

Reply:
i have added for example